frontend/scss/organisms/_o-custom-tours-landing.scss

```diff
@@ -164,4 +171,161 @@
   }
 }
 
+// -- Custom tours: Custom footer
+
+.m-cta-banner--aic-ct {
+  @include breakpoint('medium-') {
+    min-height: 240px;
+    height: fit-content;
+  }
+
+  @include breakpoint('small-') {
+    margin-top: 0;
+  }
+
+  .f-headline {
+    margin-bottom: 20px;
+
+    &::before {
+      padding-top: 0;
+    }
+
+    &::after {
+      margin-bottom: 0;
+    }
+  }
+
+  .m-cta-banner__txt {
+    padding-top: 32px;
+    padding-bottom: 32px;
+    height: fit-content;
+
+    @include breakpoint('medium+') {
+      position: absolute;
+      left: colspan(3, medium);
+      right: colspan(3, medium);
+    }
+
+    @include breakpoint('large+') {
+      left: colspan(3, large);
+      right: colspan(3, large);
+    }
+
+    @include breakpoint('xlarge') {
+      left: colspan(3, xlarge);
+      right: colspan(3, xlarge);
+    }
+  }
+
+  .m-cta-banner__action {
+    @include breakpoint('small-') {
+      position: unset;
+    }
+  }
+
+  .m-cta-banner__msg {
+    @include breakpoint('small-') {
+      margin-bottom: 20px;
+    }
+  }
+}
+
+.m-cta-banner--aic-ct--container {
+  @include breakpoint('small-') {
+    position: relative;
+    width: 100vw;
+    left: -5.2%;
+  }
+}
+
+.m-cta-banner--aic-ct__overlay {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background: linear-gradient(180deg, rgba(0, 0, 0, 0.34) 0%, rgba(0, 0, 0, 0.42) 11.17%, rgba(0, 0, 0, 0.54) 42.05%);
+}
+
+.m-cta-banner--aic-ct--with-image {
+  @include breakpoint('small-') {
+    position: relative;
+    overflow: visible;
+    overflow-y: hidden;
+
+    &::before {
```

Comment:
It's not super clear to me what this is doing – is it a full-bleed utility? Could you add a comment to explain it?